index.html

```diff
@@ -6,17 +6,174 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>My Blog</title>
+    <title>Liz life Blog</title>
     <link
       href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
       rel="stylesheet"
       type="text/css"
     />
     <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <!-- START of HOME PAGE DIV -->
+    <section >
+
+      <!-- START of NAV-BAR & HERO BANER  -->
+      <section class="topnav">
+        <nav>
+          <a href="#" style="float:right;"><i class="fa fa-search"></i
```

Comment:
The formatting is a little strange here. Sometimes the auto save formatter does strange things, but if you correct it then it doesn't change back. Here, it would be easier to read to have:
```
<a href="#" style="float:right;">
   <i class="fa fa-search"></i>
</a>
```

Reply:
Also, how does the text get rendered here?